knowledge-base/kb-restrictfirstaxis-to-chartpanewidth.md

```diff
@@ -0,0 +1,47 @@
+---
+title: How To Restrict the First Axis Label to be Within the Chart Pane Width
+description: "How to restrict the first axis label to be within the Chart Pane width of CartesianChart element."
+type: how-to
+page_title: How to restrict the first axis Label to be within the Chart Pane width of RadCartesianChart for WPF.
+slug: kb-restrictfirstaxis-to-chartpanewidth
+position: 0
+tags: axis, chart pane, label, cartesianchart
+ticketid: 1620645
+res_type: kb
+---
+
+## Environment
+
+<table>
+    <tbody>
+        <tr>
+            <td>Product Version</td>
+            <td>2023.2.718</td>
+        </tr>
+        <tr>
+            <td>Product</td>
+            <td>RadChartView for WPF</td>
+        </tr>
+    </tbody>
+</table>
+
+
+## Description
+
+How to restrict the first axis label to be within the `Chart Pane` width of `RadCartesianChart` element.
```

Comment:
There is no such thing as Chart Pane. Don't use capital letters. Or the `special formatting` which is reserved only for API names - class, property, methods and others.
It is good to use the "chart pane" in the article because this is what the user said, but I suggest to keep the correct terminology where possible. In the Description section of the article, you can use "plot area" instead of "chart pane" and add a link to the article for the chart's Plot Area.